redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -1,83 +1,101 @@
+      name: t('footer.contact')
+    },
+    {
+      url: 'https://www.redi-school.org',
+      name: t('footer.name')
+    }
+  ]
+
+  const legalLinks = [
+    {
+      url: 'https://www.redi-school.org/berlin-transparency/',
+      name: t('footer.transparency')
+    },
+    {
+      url: 'https://www.redi-school.org/imprint/',
+      name: t('footer.cookies')
+    },
+    {
+      url: 'https://www.redi-school.org/data-privacy-policy',
+      name: t('footer.privacy')
+    }
+  ]
+
   return (
-    <footer className="footer">
+    <footer className="footer footer--border">
       <Section>
         <Container>
-          <Columns>
-            <Columns.Column className="is-hidden-mobile">
-              <p>
-                <a href="https://www.redi-school.org/">{t('footer.name')}</a>
-              </p>
-              <p>&copy; {year} {t('footer.copyright')}</p>
-            </Columns.Column>
-            <Columns.Column className="is-hidden-tablet footer--headline">
-              <p className="is-size-5">{t('footer.socialMediaHeadline')}</p>
-              <Icons />
+          <Columns breakpoint="mobile">
+            <Columns.Column mobile={{ size: 'full' }}>
+              <Element renderAs="a" href="https://www.redi-school.org" target="_blank">
+                <RediSchool className="oneandhalf-bs" />
+              </Element>
+              <Element renderAs="a" href="https://www.deloitte.de" target="_blank">
+                <Deloitte />
+              </Element>
             </Columns.Column>
-            <Columns.Column className="footer--links">
-              <p>
-                <a
-                  href="https://www.redi-school.org/imprint"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.contact')}
-                </a>
-              </p>
-              <p>
-                <a href="/" target="_blank" rel="noopener noreferrer">
-                  {t('footer.faq')}
-                </a>
-              </p>
-              <p>
-                <a
-                  href="https://www.redi-school.org/berlin-transparency/"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.transparency')}
-                </a>
-              </p>
-              <p>
-                <a href="/">{t('footer.cookies')}</a>
-              </p>
-              <p>
-                <a
-                  href="https://www.redi-school.org/data-privacy-policy"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.dataPolicy')}
-                </a>
-              </p>
+            <Columns.Column mobile={{ offset: 'zero', size: 'half' }} desktop={{ offset: 1 }} className="double-bs">
+              <Element renderAs="ul">
+                <Element renderAs="li" textSize={6} textTransform="uppercase" textWeight="bold">
```

Comment:
Semantically this is not the part of the list, it's the heading of the same. 
Please keep it apart. 

Also, you can attempt to reuse the already defined heading separator instead of the HR solution.